Dockerfile.assisted-installer

```diff
@@ -1,16 +1,20 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.20 AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20 AS builder
 ARG TARGETPLATFORM
 ENV GOFLAGS=-mod=mod
 WORKDIR /go/src/github.com/openshift/assisted-installer
 
+# Workaround for creating build folder
```

Comment:
Why using a workaround when we can use this image the way it's meant to?

Reply:
It's not being used correctly on master?

https://github.com/openshift/assisted-installer/blob/master/Dockerfile.assisted-installer#L7